File: js/tests/sunspider-1.0/c,c++/bitops-bits-in-byte.c
int bitsinbyte(int b){
    int m = 1, c = 0;
    while(m<0x100) {
        if(b & m) c++;
        m <<= 1;
    }
    return c;
}

int main() {
    int x, y, t;
    int sum = 0;
    for(x=0; x<350; x++){
        for(y=0; y<256; y++){
         sum += bitsinbyte(y);
        }
    }
    return sum;
}

File: js/tests/sunspider-1.0/ts/bitops-bits-in-byte.ts
function bitsinbyte(b: int): int{
    let m: int = 1, c = 0;
    while(m<0x100) {
        if(b & m) c++;
        m <<= 1;
    }
    return c;
}

export function main(): int {
    let x: int, y, t;
    let sum: int = 0;
    for(let x: int=0; x<350; x++){
        for(let y: int=0; y<256; y++){
         sum += bitsinbyte(y);
        }
    }
    return sum;
}
